docs/user-guide/horizontal-pod-autoscaler.md

```diff
@@ -7,7 +7,6 @@ This document describes the current state of Horizontal Pod Autoscaler in Kubern
 
 Horizontal pod autoscaling allows the number of pods in a replication controller or deployment
```

Comment:
I think it sounds better:

"HPA allows to automatically scale the number pods in ... based on observed CPU utilization"